wremnants/syst_tools.py

```diff
@@ -301,24 +302,24 @@ def add_pdf_hists(results, df, dataset, axes, cols, pdfs, base_name="nominal", a
             logger.warning(f"PDF {pdf} was not found for sample {dataset}. Skipping uncertainty hist!")
             continue
 
+        pdfHist = df.HistoBoost(pdfHistName, axes, [*cols, tensorName], tensor_axes=[pdf_ax], storage=hist.storage.Weight())
         if pdfInfo["alphasRange"] == "001":
-            alphaSHistName = Datagroups.histName(base_name, syst=f"{pdfName}alphaS001")
+            name = Datagroups.histName(base_name, syst=f"{pdfName}alphaS001")
```

Comment:
I think it doesn't hurt to keep the variable's name as "alphaSHistName", given that the PDFs get their name from "pdfHistName".

Also, even though this was actually part of the previous code, instead of having "if pdfInfo['alphasRange'] == '001' " to build the name of the alphaS histogram, perhaps one may direct form the string using 
alphaSHistName = Datagroups.histName(base_name, syst=f"{pdfName}alphaS{pdfInfo['alphasRange']}")